systemd/emulation.go

```diff
@@ -188,11 +201,31 @@ func (s *emulation) RemoveMountUnitFile(mountedDir string) error {
 		}
 	}
 
-	multiUserTargetWantsDir := filepath.Join(dirs.SnapServicesDir, "multi-user.target.wants")
-	enableUnitPathSymlink := filepath.Join(multiUserTargetWantsDir, filepath.Base(unit))
-	if err := os.Remove(enableUnitPathSymlink); err != nil {
+	// `systemctl disable` removes all symlinks
```

Comment:
It's not clear to me why this comment is here, since we are not using systemctl.

Reply:
We emulate  it `systemctl disable`, so we have to do the same. I have rewritten it to "`systemctl disable` removes all symlinks, so we should do the same". Hope this is clearer.